### DISCUSSION TITLE
Ability to rename a file. I can create a new one and can edit its contents but I cannot rename/edit the file/extension.

### DISCUSSION THREAD

**Discussion by AndrewHaglund** [community/community#108894] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.149.0 (122782244)
Device: iPhone 15 Pro Max (iOS 17.3.1)
</details>

**Comment by iskae** [comment]
Hey @AndrewHaglund 👋 

Thank you for reaching out ❤️ We have already an internal tracking issue for this 🎉 I will be adding your request in there too 🙇 

Please stay tuned for our future releases 😊
